test/runnable/b17073.d

```diff
@@ -0,0 +1,13 @@
+struct S0
```

Comment:
Since the test case is limited to this file I don't think the name really matters